modules/ComscoreStreamingTag/ComscoreStreamingTag.manifest.php

```diff
@@ -0,0 +1,72 @@
+<?php 
+/**
+ * The ComScoreStreamingTag plugin manifest
+ */
+
+return array(
+	'ComScoreStreamingTag' => array(
+		'description' => "Supports sending Streaming tag player analytics events to comScore.",
+		'label' => "comScoreStreamingTag",
+		'attributes'=> array(
+			'c2' => array(
+				'doc' => "Used to specify the comScore client ID (also called 'Customer C2') which identifies the comScore account for which the data should be collected. Expects a number with 7 or 8 digit.",
+				'label' => 'comScore client ID',
+				'type' => 'String',
+			),
+
+			'persistentLabels' => array(
+				'doc' => "Advanced configuration setting! Used to specify 'persistent labels' which are added to all measurements. Expects a comma-separated list of 'name=value' assignments. Please do not use this setting unless you are explicitly instructed to do so by your comScore client account representative.",

```

Comment:
should also mention evaluations being supported, per code review comment if possible you should do late evaluates of this.